comps/comps-foreman-el8.xml

```diff
@@ -257,6 +257,7 @@
       <packagereq type="default">rubygem-prometheus-client</packagereq>
       <packagereq type="default">rubygem-promise.rb</packagereq>
       <packagereq type="default">rubygem-public_suffix</packagereq>
+      <packagereq type="default">rubygem-pulp_python_client</packagereq>
```

Comment:
I'd think these need to go to katello comps? :)

Reply:
Oh, I missed that. I used `./add_gem_package.sh pulp_python_client:3.2.0 scl katello-nightly-rhel7`. Maybe there's a bug with the script.

Reply:
It should. @ekohl did some changes recently, maybe that backfired?